src/content/recipes/styled-components.md

```diff
@@ -220,10 +266,10 @@ export const parameters = {
   },
 };
 
-export const decorators = [withTheme];
+export const decorators = [withGlobalStyle, withTheme];
```

Comment:
If we flip the order here, you can then access `theme` inside `GlobalStyle`. Perhaps we should demonstrate that, too? Or maybe just a comment explaining why the order matters? Something like this:

```suggestion
// Decorators run in the order they are provided. By placing `withTheme` before `withGlobalStyle`, the GlobalStyle component can access the `theme` via context
export const decorators = [withTheme, withGlobalStyle];
```

Reply:
I think describing this behavior may be too nuanced for a basic recipe, especially given that our `withGlobalStyle` doesn't use `theme`.

Reply:
That's fair. I was concerned about that balance, too. Let's at least not set them up for failure and flip the order?

```suggestion
export const decorators = [withTheme, withGlobalStyle];
```

Reply:
I thought that they were applied left to right? withGlobalStyles then withTheme would give globaltyles access to the theme